com.unity.ml-agents/Runtime/Inference/BarracudaModelParamLoader.cs

```diff
@@ -19,8 +19,9 @@ internal enum ModelApiVersion
         {
             MLAgents1_0 = 2,
             MLAgents2_0 = 3,
+            MLAgents2_0_Recurrent = 4,
```

Comment:
Since we haven't released a version that uses `MLAgents2_0`, I'd be in favor of just defining `MLAgents2_0` to contain the LSTM changes too.

Either way, we should describe the differences between the versions here, just for our own internal knowledge.

Reply:
Yeah, you might consider calling `MLAgents2_0` => `MLAgents1_9` if that is the model version that went out with the last release.